src/class_singleWorker.py

```diff
@@ -1120,7 +1120,7 @@ def sendMsg(self):
                 privSigningKeyHex, privEncryptionKeyHex, \
                     pubSigningKey, pubEncryptionKey = self._getKeysForAddress(
                         fromaddress)
-            except:
+            except:  # noqa:E722
```

Comment:
same, `KeyError` feedback to user, general error log warning

Reply:
Updated